openmetadata-service/src/main/java/org/openmetadata/service/resources/glossary/GlossaryTermResource.java

```diff
@@ -244,6 +244,7 @@ public ResultList<GlossaryTerm> list(
     } else { // Forward paging or first page
       terms = repository.listAfter(uriInfo, fields, filter, limitParam, after);
     }
+    repository.addChildrenCount(terms);
```

Comment:
we should do it in setFields of glossaryTerms I feel, this should not be be handled in the listing api